pkgs/development/libraries/mlt/qt-5.nix

```diff
@@ -33,7 +35,8 @@ mkDerivation rec {
     owner = "mltframework";
     repo = "mlt";
     rev = "v${version}";
-    sha256 = "sha256-Y7lbfwA0lkQB3PjYQIQaQ0BeXGcgyCmMnDqqZJ8zUaA=";
+    sha256 = "sha256-rio8HpSa3CpyyrrbaaEmHzM+xJgCl1CCAlVs/9IXq9U=";
+    fetchSubmodules = true; # For now only contains glaxnimate… not sure how/if it's possible to use directly the packaged glaxnimate.
```

Comment:
What happens if you don't do this?

Reply:
It fails to find some `*.in.hpp` files downloaded by the submodule.

Reply:
Can we symlink our glaxnimate to that location instead?

Reply:
You mean the final derivation? If yes, no because glaxnimate's output derivation is just a binary + some share files (icons, plugins, python scripts…), while here it tries to use internal functionality of glaxnimate that are not exposed as libraries.

Reply:
can you use `glaxnimate.src` then?

Reply:
I have to admit that I don't see much advantage of using `glaxnimate.src` as an upgrade to glaxnimate could easily break the current package, for instance if glaxnimate removes or rename the C functions used in this plugin… (which will certainly arrive as they plan to refactor the code of the library) And I don't see much advantage over fetching the module, anI missing something?

Reply:
I think fetching it as a submodule is fine as long as it isn't packaged better.